documentation/API-design-guidelines.md

```diff
@@ -1287,44 +1288,51 @@ response:
+Note: If the API provides both patterns (indirect and resource-based), and the API customer requests both (instance based + subscription), the 2 requests should be handled independently & autonomously. Depending on server implementation, it is acceptable, when the event occurs, that one or two notifications are sent to listener.
 
 
 ### 12.2 Event notification
 
 The event notification endpoint is used by the API server to notify the API consumer that an event occured.
 
-Note: The notification is the message posted on listener side. We describe the notification message in the CAMARA OAS but it could confusing because this endpoint should be implemented on the business API consumer side. This notice should be explicited mentioned in all CAMARA API documentation featuring notifications.
+CAMARA event notification leverages **CloudEvents** as it is a vendor-neutral specification for defining the format of event data. A generic neutral CloudEvent notification swagger is available in commonalities/artifact directory (notification-as-cloud-event.yaml).
+
+Note: The notification is the message posted on listener side. We describe the notification(s) in the CAMARA API using the `callbacks`. From API consumer it could confusing because this endpoint must be implemented on the business API consumer side. This notice should be explicited mentioned in all CAMARA API documentation featuring notifications.
 
-Only Operation POST is provided for `eventNotifications` and the expected response code is `204`.
+Only Operation POST is provided for `eventNotifications` and the expected response code is `204`. 
+The url for this `POST` operation must be specified in the swagger as `{$request.body#/webhook/notificationUrl}`. 
 
-For consistence among CAMARA APIs the uniform `eventNotifications` model must be used:
+For consistence among CAMARA APIs the uniform CloudEvents model must be used with following rules:
 
 | name | type | attribute description | cardinality |
 | ----- |	-----  |	 -----  |  -----  | 
-| eventSubscriptionId | string | subscription identifier - could be valued for Resource-based subscription | optional |
-| event | object | event structure - see next table | mandatory |
+| id | string | identifier of this event, that must be unique in the source context. | mandatory |
+| source | string - URI | identifies the context in which an event happened in the specific Provider Implementation. Often this will include information such as the type of the event source, the organization publishing the event or the process that produced the event. The exact syntax and semantics behind the data encoded in the URI is defined by the event producer. | mandatory |
+| type | string | a value describing the type of event related to the originating occurrence. For consistency accross API we mandate following pattern: org.camara.api._name of the API_._event name_ (for exemple org.camara.api.device-status.RoamingStatusEvent ) | mandatory |
```

Comment:
Taking into account this https://github.com/camaraproject/Commonalities/pull/43#discussion_r1319631841 and the convention for DNS and reverse-DNS names to use lowercase :
```suggestion
| type | string | a value describing the type of event related to the originating occurrence. For consistency accross CAMARA APIs we mandate following pattern of reverse-DNS written in lowercase: org.camara.api._name of the API_._event name_ (for exemple org.camara.api.device-status.roamingstatusevent ) | mandatory |
```

Reply:
The option is to use more readable kebab-case i.e.  `org.camara.api.device-status.roaming-status-event`

Reply:
> The option is to use more readable kebab-case i.e. `org.camara.api.device-status.roaming-status-event`

Hi @PedroDiez, @rartych, @bigludo7 
Sorry, I'm not sure to understand  you well.
IMHO I thought here we are talking about the fully qualified class name and not the DNS FQDN and that is why we can use capital letters.
This is confusing because the package name mimics the reverse DNS name.
If I'm wrong, please feel free to correct me.

Reply:
Please see my comment below: https://github.com/camaraproject/Commonalities/pull/56#issuecomment-1721337366